doc/SLSKPROTOCOL.md

```diff
@@ -67,25 +67,25 @@ If you find any inconsistencies, errors or omissions in the documentation, pleas
 | 1    | Away    |
 | 2    | Online  |
 
-### Transfer Direction
+### Transfer Directions
 
-| Code | Type     |
-| ---- | -------- |
-| 0    | Download |
-| 1    | Upload   |
+| Code | Direction          |
+| ---- | ------------------ |
+| 0    | Download from Peer |
+| 1    | Upload to Peer     |
 
 ### File Attribute Types
 
-| Code | Type             |
-| ---- | ---------------- |
-| 0    | Bitrate (kbps)   |
-| 1    | Length (seconds) |
-| 2    | VBR (0 or 1)     |
-| 3    | Encoder (unused) |
-| 4    | Sample Rate (Hz) |
-| 5    | Bit Depth        |
+| Code | Value             |
+| ---- | ----------------- |
+| 0    | Bitrate (kbps)    |
+| 1    | Length (seconds)  |
+| 2    | VBR (0 or 1)      |
+| 3    | Encoder (unused)  |
+| 4    | Sample Rate (Hz)  |
+| 5    | Bit Depth (bits)  |
 
-#### Used Attribute Combinations
+#### File Attribute Combinations
```

Comment:
The title needs to emphasize that these combinations are actively used by clients. More combinations are possible, of course.

Reply:
"`Known`" ?